boards/cubepilot/cubeyellow/test.cmake

```diff
@@ -3,7 +3,7 @@ px4_add_board(
 	PLATFORM nuttx
 	VENDOR cubepilot
 	MODEL cubeyellow
-	LABEL console
+	LABEL test
```

Comment:
Unrelated: is there a reason we can't just take the label from the file name?

Reply:
No reason that immediately comes to mind. I think that would be a nice simplification.